### ISSUE TITLE
First Android app codelab: textAlignment not found in part 6, step 10, number 2.

### ISSUE THREAD

**Issue by damienscarpellini** [google-developer-training/first-android-app#114]
![image](https://user-images.githubusercontent.com/77271505/123632657-177bd580-d810-11eb-9560-6973b32f4231.png)

Under Step 10. part 2. I couldn't find textAlignment, however after a quick google search, it was recommended to use gravity center instead.

If textAlignment exists, might be worth describing where to find it.

**Comment by rajgupta93** [comment]
Yes, you are absolutely right to make text alignment center, left or right. We use attribute gravity.
